addons/website_slides/models/slide_question.py

```diff
@@ -27,19 +27,19 @@ def _check_only_one_good_answer(self):
                 if answer.is_correct:
                     good_answer_count += 1
                     if good_answer_count > 1:
-                        raise ValidationError(_('A question can only have one good answer'))
+                        raise ValidationError(_('The question "%s" can only have one good answer') % question.question)
```

Comment:
You may remove "The" in all those errors. "Question Youplaboum can only have one good answer" is better than "The question ..." . The is too much used by french speaking people.